src/wasm/kotlin/gradle.properties

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2018 JetBrains s.r.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+library.version=0.11.0
+kotlin.version=1.3.30
+kotlin.version.snapshot=1.3-SNAPSHOT
+plugin.version=1.3.30
+plugin.version.snapshot=1.3-SNAPSHOT
+konan.version=1.3.30
+konan.version.snapshot=1.3-SNAPSHOT-native-1.2.0-dev
+# Also set KONAN_LOCAL_DIST enviroment variable to auto-assign konan.home
+
+native.deploy=false
+
+org.gradle.parallel=true
+org.gradle.caching=false
+
+org.jetbrains.kotlin.native.jvmArgs=-XX:TieredStopAtLevel=1
+#konan.jvmArgs=-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5007
```

Comment:
When might one need to use this?

Reply:
If you want to debug the Kotlin->WASM compiler itself I guess. It's from Kotlin's samples.